packages/Input/src/Input.js

```diff
@@ -153,7 +186,7 @@ const defaultProps = {
   a11yText: null,
   clearIcon: null,
   className: null,
-  defaultValue: "",
+  defaultValue: null,
```

Comment:
Slight change to the API, but I don't think this constitutes a breaking change.

Reply:
🤔 but `null` value yields that inputs need a string value, wouldn't be better `undefined` ?

Reply:
Okay, I see we do sometimes use `undefined` in `defaultProps`, so I can change this.